packages/data-context/src/actions/EventCollectorActions.ts

```diff
@@ -1,6 +1,8 @@
 import type { DataContext } from '..'
 import Debug from 'debug'
 
+const pkg = require('@packages/root')
```

Comment:
I think this is already maintained in the context, so this may be duplicative. Could you do something like:

`'x-cypress-version': (await this.ctx.versions.versionData()).current.version`?